build.gradle

```diff
@@ -59,6 +63,27 @@ task createBalaDirectory {
     mkdir "${project.projectDir}/sfdc/target/bala"
 }
 
+//task sfdcTestBuild {
+//    exec {
+//        commandLine 'sh', '-c', "${ballerinaDistributionPath}/bin/bal build -c --skip-tests ./sfdc"
+//    }
+//    inputs.dir file(project.projectDir)
+//
+//    pathToBala.eachFileMatch(~/.*.bala/) { balaFile ->
+//        copy {
+//            from zipTree(balaFile)
+//            into file("${artifactCacheParent}/bala/${packageOrg}/${packageName}/${project.version}/${platform}")
+//        }
+//    }
+//    copy {
+//        from file("${project.projectDir}/sfdc/target/cache")
+//        exclude '**/*-testable.jar'
+//        exclude '**/tests_cache/'
+//        into file("${artifactCacheParent}/cache/")
+//    }
+//    outputs.dir artifactCacheParent
+//}
```

Comment:
Why is it commented?
Do we need this task?

Reply:
Removed this one it was unnecessary